git_machete/cmd.py

```diff
@@ -3190,7 +3269,7 @@ def usage(c: str = None) -> None:
 
               <b>-l, --list-commits</b>           When printing the status, additionally list the messages of commits introduced on each branch.
 
-              <b>-M, --merge</b>                  Update by merge rather than by rebase.
+              <b>-M, --merge</b>                  Update, and detect updates, by merge rather than by rebase.
```

Comment:
If you could expand this explanation a bit, so that the cases of 1) update 2) detection are considered separately

Reply:
I know it's described above in docs for `traverse` itself... but still, sth similar to the above docs of `--merge` option of `status` would be useful